recipes/openmp/bld.bat

```diff
@@ -0,0 +1,20 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+    -DCMAKE_BUILD_TYPE="Release" ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX:PATH=%LIBRARY_PREFIX% ^
+    %SRC_DIR%
+
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
+nmake install
+if errorlevel 1 exit 1
+
+mkdir -p %LIBRARY_PREFIX%/lib/clang/%PKG_VERSION%/include
```

Comment:
Unfortunately `mkdir -p` is not a thing on Windows. Though you could do `if not exist "<dir>" mkdir <dir>`, which will do the same thing. Please see this [SO answer]( https://stackoverflow.com/a/20688004 ) for more details.

Also please add `if errorlevel 1 exit 1` after this line.